src/test/java/ApiTests.java

```diff
@@ -0,0 +1,27 @@
+import io.qameta.allure.Owner;
+import models.Root;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static helpers.ConfigSetUp.*;
```

Comment:
It's better to import only necessary classes.

Reply:
done